validator/sawtooth_validator/execution/scheduler_parallel.py

```diff
@@ -65,10 +65,10 @@ def __repr__(self):
         return repr(self._root)
 
     def _tokenize_address(self, address):
-        return [
+        return (
             address[i:i + self._token_size]
             for i in range(0, len(address), self._token_size)
-        ]
+        )
```

Comment:
This change should probably be in a separate commit.

Reply:
I don't think it's worth a full commit, but I added a comment about it in the `get_ancestors` commit.